repository/m.json

```diff
@@ -277,7 +277,7 @@
 		{
 			"name": "MarkdownEditing",
 			"details": "https://github.com/SublimeText-Markdown/MarkdownEditing",
-			"labels": ["markdown", "github flavored markdown", "github markdown", "gfm", "multimarkdown", "color scheme"],
```

Comment:
Hm okay, `github flavored markdown` provides roughly the same information as `github markdown` as a label, so we only need one. Technically the former is the correct one, but the latter is shorter.